deltachat-repl/src/main.rs

```diff
@@ -317,7 +317,7 @@ async fn start(args: Vec<String>) -> Result<(), Error> {
         .await?;
 
     let events = context.get_event_emitter();
-    tokio::task::spawn(async move {
+    spawn_named_task!("repl:receive_event", async move {
```

Comment:
Jsonrpc tasks aren't prefixed, so maybe don't prefix anything at all? Having Location in the console seems sufficient